socorro/scripts/pubsub.py

```diff
@@ -142,7 +142,10 @@ def publish_crashid(config, args):
     topic_name = config['resource.pubsub.%s_topic_name' % queue]
     topic_path = publisher.topic_path(project_id, topic_name)
 
-    for crash_id in args.crashid:
+    if not args.crashid:
+        print('Reading crash IDs from stdin')
```

Comment:
You don't need to change this, but generally, I think we refer to it as "crash id" or "crashid" all lowercase.